mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -164,3 +170,137 @@ def _convert_mm_to_m(array):
 
 def _convert_deg_to_rad(array):
     return array * np.pi / 180.0
+
+
+def convert_units(inst, calibration, to="radians"):
+    """Convert Eyegaze data from pixels to radians of visual angle or vice versa.
+
+    Parameters
+    ----------
+    inst : instance of Raw, Epochs, or Evoked
+        The Raw, Epochs, or Evoked instance with eyegaze channels.
+    calibration : Calibration
+        Instance of  Calibration, containing information about the screen size
+        (in meters), viewing distance (in meters), and the screen resolution
+        (in pixels).
+    to : str
+        Must be either ``"radians"`` or ``"pixels"``, indicating the desired unit.
+
+    Returns
+    -------
+    inst : instance of Raw | Epochs | Evoked
+        The Raw, Epochs, or Evoked instance, modified in place.
+
+    Notes
+    -----
```

Comment:
Here I think you need some text about the caveat that the conversion assumes linearity and it's only approximately linear, taken/rephrased from your comment would be good